shenyu-plugin/shenyu-plugin-base/src/test/java/org/apache/shenyu/plugin/base/condition/judge/PredicateJudgeFactoryTest.java

```diff
@@ -51,7 +59,7 @@ public void testConditionDataIsNull() {
 
     @Test
     public void testRealDataIsBlank() {
-        assertFalse(PredicateJudgeFactory.judge(conditionData, null));
```

Comment:
I think this line can be kept. Check whether there are other `null test` lines that can be kept.